whois-update/src/main/java/net/ripe/db/whois/update/handler/validator/domain/NServerValidator.java

```diff
@@ -45,7 +47,10 @@ public void validate(final PreparedUpdate update, final UpdateContext updateCont
                 case IP6:
                 {
                     final boolean endsWithDomain = domain.endsWithDomain(nServer.getHostname());
-
+                    if (domain.getReverseIp() != null) {
+                        validateRipeNsServerPrefixLength(domain, update, domain.getType().equals(INADDR),
+                                nServerAttribute, updateContext);
```

Comment:
Suggestion: If you are passing the domain object anyways  you do not need to pass a isIpv4 boolean? 
You can have this check in the method itself. It makes it more readable